tests/test_auth.py

```diff
@@ -0,0 +1,81 @@
+import json
+import unittest
+from flask import Flask
+
+from app import create_app, db
+from app.models import User
+
+class BaseTest(unittest.TestCase):
```

Comment:
Since you have defined a `BaseTest` class in `tests/test_resources.py`, you don't need to have it defined here again

A wise lady once said 
> Base test to be used by each test method. It enables the DRY principles to be followed. Dee (2017)